src/clr-core/styles/tokens/tokens.stories.mdx

```diff
@@ -0,0 +1,85 @@
+import { Meta, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
+<Meta title="Experimental/Design Tokens/Getting Started" />
+
+# Design Tokens
+
+<cds-alert status="danger" closable="false" cds-layout="m-b:md">
+  <cds-alert-content>This is a experimental API and will change. Do not use this in production enviroments!</cds-alert-content>
+</cds-alert>
+
+Design Tokens are global variables to configure the foundation of the
+design system. Changes to a design token will propagate throughout the entire system.
+Tokens are broken down into five categories:
+
+- [Spacing](./?path=/docs/experimental-spacing-getting-started--page)
+- [Layout](./?path=/docs/experimental-layout-get-started--page)
+- [Typography](./?path=/docs/experimental-typography-getting-started--page)
+- [Color](./?path=/docs/experimental-color-getting-started--page)
+- Global
+
+## Installation
+
+To use the design token system via CSS import the global CSS file.
```

Comment:
`To use the design token system via CSS, import ` not `To use the design token system via CSS import `